instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './SearchBar.css';
+import person from './person.jpg';
+import heart from './heart.png';
+import logo from './instagram-logo.png';
+import text from './instagram-text-logo.png';
+import diamond from './circle-diamond.png';
+
+class SearchBar extends React.Component {
```

Comment:
If you don't have any state or lifecycle methods, why not use a functional component? That is, unless you were planning on using the controlled input pattern to keep track of what's inside your input tag